src/services/user/index.js

```diff
@@ -59,6 +59,12 @@ module.exports = (app) => {
 	const firstLoginService = app.service('firstLogin');
 	firstLoginService.hooks(firstLoginHooks);
 
+	// eslint-disable-next-line global-require
+	const ForcePasswordChangeService = require('./services/forcePasswordChange')(app);
+	app.use('/forcePasswordChange', new ForcePasswordChangeService());
+	const forcePasswordChangeService = app.service('forcePasswordChange');
+	forcePasswordChangeService.hooks(firstLoginHooks);
```

Comment:
Over firstLoginHooks do you have no protection. Every user and role with jwt can execute this. But only superhero and the current user params.account.userId should execute it.

Reply:
Edit and person with permission to do this at the same school. (schoolId is in user doc)

Reply:
According to our discussion no needed to do that as it is secured by jwt